apps/design-land/src/app/image/image.component.html

```diff
@@ -1 +1,28 @@
-<daff-image src="/assets/card/uber-logo.svg" alt="uber logo" width="300" height="119"></daff-image>
\ No newline at end of file
+<daff-article>
+  <h1 daffArticleTitle>Image</h1>
+  <p daffArticleLead>
+    The image component is used to customize an image you want to display in your project. To those familiar with 
+    <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img" target="_blank">the HTML <code>img</code></a>, 
+    <code>daff-image</code> should feel similar. <code>daff-image</code> is an opinionated version that encourages end-user friendly usage.
+  </p>
+
+  <h3>Usage</h3>
```

Comment:
Usage and size can be replaced with:

```
<h2>Attributes</2>
The `src`, `width`, `height`, and `alt` attributes must be defined. An error will be thrown any of these attributes are missing.

`width` and `height` are required to calculate the aspect ratio of an image, used for the [aspect ratio padding trick](https://css-tricks.com/aspect-ratio-boxes/) that helps to prevent content jumping while images are loading. The `width` and `height` values are rendered as pixels.
```